src/main/java/com/fasterxml/jackson/core/JsonParser.java

```diff
@@ -193,6 +193,12 @@ public enum Feature {
         @Deprecated
         ALLOW_UNQUOTED_CONTROL_CHARS(false),
 
+        /**
+         * @deprecated Use {@link com.fasterxml.jackson.core.json.JsonReadFeature#ALLOW_RS_CONTROL_CHAR} instead
```

Comment:
May look funny but `JsonParser.Feature` is to be removed from 3.0 yet is needed internally (all `StreamReadFeature`s and `JsonReadFeature`s must map back to one entry).
But we don't want users to use `JsonParser.Feature` any more.

Reply:
It's good to know about that to help take care of it next time. 
Thanks for your collaboration on the PRs!